pyconcz/templates/default/proposals/proposal_before.html

```diff
@@ -5,7 +5,7 @@
 {% block meta_title %}Call for Papers opens soon – {{ block.super }}{% endblock %}
 
 
-{% block content %}
+{% block main %}
```

Comment:
Good catch! But, because it has nothing in common with email, move it please to separated commit.